pages/opengraph/collection/[collectionId].tsx

```diff
@@ -0,0 +1,42 @@
+import { useRouter } from 'next/router';
+import { OpenGraphPreview } from 'components/opengraph/OpenGraphPreview';
+import useCollectionById from 'hooks/api/collections/useCollectionById';
+
+export default function OpenGraphCollectionPage() {
+  const { query } = useRouter();
+  const collection = useCollectionById({ id: query.collectionId as string });
+
+  const width = parseInt(query.width as string) || 600;
+  const height = parseInt(query.height as string) || 300;
+
+  if (!collection) {
+    // TODO: render something nice?
+    throw new Error('no collection found');
```

Comment:
in these cases we should just fall back to the default image configured in `app`

Reply:
agreed, we can show what we currently show in our link previews